src/main/java/uk/gov/hmcts/reform/jobscheduler/services/JobsService.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.jobscheduler.services;
+
+import org.apache.commons.lang.NotImplementedException;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.jobscheduler.model.Job;
+
+@Service
+public class JobsService {
+
+    public String create(Job job) {
```

Comment:
Worth considering writing a test for this method, so that when someone implements it, it will start failing. I'm not strong about it, but it will reduce the risk of missing the tests.